templates/cis2-nft/src/lib.rs

```diff
@@ -0,0 +1,1061 @@
+//! Note: This contract is not meant for production, it is used to illustrate
+//! how to use the standard library and the tooling Concordium provides. There
+//! is no claim that the logic of the contract is reasonable, or safe.
```

Comment:
Do we want to have this kind of thing in a template we advertise as a cis2-nft?

It seems to me that people will use this if we make it easy, like we are doing. Do we want to revise this disclaimer, or what should we do?

Reply:
Since this disclaimer is part of the template, we can revise/change it at any time. We could start with this disclaimer and once the wCCD token contract is security audited (+ the cis2 library), we have higher confidence that we can remove the disclaimer. Internally, we could security audit the wCCD as well as the cis2-nft token contracts if both of them are used most likely in production.

Reply:
That is a good point.